testsuite/features/support/commonlib.rb

```diff
@@ -168,14 +168,6 @@ def find_and_wait_click(*args, **options, &optional_filter_block)
   element.extend(CapybaraNodeElementExtension)
 end
 
-def get_client_type(name)
-  if name.include? '_client'
-    'traditional'
-  else
-    'salt'
-  end
-end
-
```

Comment:
I wonder where this method is used throughout our code. Did you had a look because I did not find anything in this PR regarding replacing it.

Reply:
it's used there:
```
testsuite/features/step_definitions/command_steps.rb:  if get_client_type(host) == 'traditional'
testsuite/features/step_definitions/common_steps.rb:  if get_client_type(host) == 'traditional' and is_salt.empty?
```

Reply:
Which was also removed. So all good.